network-store-client/src/main/java/com/powsybl/network/store/client/RestClient.java

```diff
@@ -23,6 +24,8 @@ public interface RestClient {
 
     <T extends IdentifiableAttributes> Optional<Resource<T>> getOne(String target, String url, Object... uriVariables);
 
+    Optional<ExtensionAttributes> getOneExtensionAttributes(String url, Object... uriVariables);
```

Comment:
Add comment saying that we don't need Resource here, only Attributes and why

Reply:
Tried to add a comment in https://github.com/powsybl/powsybl-network-store/pull/411/commits/3638f11e85c23f0e44971390f11a7093d01142ec but not sure it makes it clear.